src/types/models/index.ts

```diff
@@ -180,6 +180,16 @@ export interface SearchCallLead {
   duration: number
 }
 
+export interface SearchWhatsappLead {
```

Comment:
Would it make sense to extract common parts of Message, Call and WhatsApp leads to a shared interface?